conda/libmesh-vtk/build.sh

```diff
@@ -19,15 +19,16 @@ cmake .. -G "Ninja" \
     -DCMAKE_PREFIX_PATH:PATH=${VTK_PREFIX} \
     -DCMAKE_INSTALL_PREFIX:PATH=${VTK_PREFIX} \
     -DCMAKE_INSTALL_RPATH:PATH=${VTK_PREFIX}/lib \
-    -DBUILD_DOCUMENTATION:BOOL=OFF \
-    -DBUILD_TESTING:BOOL=OFF \
-    -DBUILD_EXAMPLES:BOOL=OFF \
+    -DVTK_BUILD_DOCUMENTATION:BOOL=OFF \
+    -DVTK_BUILD_TESTING:BOOL=OFF \
+    -DVTK_BUILD_EXAMPLES:BOOL=OFF \
     -DBUILD_SHARED_LIBS:BOOL=ON \
-    -DVTK_Group_MPI:BOOL=ON \
-    -DVTK_Group_Rendering:BOOL=OFF \
-    -DVTK_Group_Qt:BOOL=OFF \
-    -DVTK_Group_Views:BOOL=OFF \
-    -DVTK_Group_Web:BOOL=OFF \
+    -DVTK_USE_MPI:BOOL=ON \
+    -DVTK_GROUP_ENABLE_MPI:STRING=YES \
+    -DVTK_GROUP_ENABLE_Rendering:STRING=YES \
```

Comment:
FYI, this change was necessitated by a new VTK module dependency. Here's the Cmake output if `Rendering` is set to `NO`:
```
CMake Error at CMake/vtkModule.cmake:1022 (message):
  The VTK::IOMPIParallel module (enabled via a `YES` setting (via
  `VTK_GROUP_ENABLE_MPI`)) requires the disabled module VTK::IOGeometry
  (disabled due to the VTK::FiltersHybrid module not being available (due to
  the VTK::RenderingCore module not being available (via a `NO` setting (via
  `VTK_GROUP_ENABLE_Rendering`)))).
Call Stack (most recent call first):
  CMakeLists.txt:303 (vtk_module_scan)
  ```

Reply:
`moosebuild` seems to appreciate `DONT_WANT` for this instead, which only builds aspects of the module that are strictly needed. Something about building the entire module with these configuration settings generates a mysterious error.